package.json

```diff
@@ -14,5 +14,8 @@
   "main": "./index.js",
   "engines": {
     "node": ">= 0.6.0"
+  },
+  "devDependencies": {
+    "tape": "^3.5.0"
```

Comment:
please use mocha for tests, and the dependency version must be specified explicitly